docs/installation.md

```diff
@@ -0,0 +1,47 @@
+## Wordpress Installation using Boiler Plate
+1. Clone this repository.
+```sh
+git clone https://github.com/ColoredCow/wordpress-init.git
+```
```

Comment:
After cloning this repo, there are few more steps:
1. Change remote URL to your project github URL - add command
2. Change the theme name to your project theme name
3. Update .gitignore with the new theme name
4. Push the code to your github project repo
5. Then the composer install command

Reply:
Think about how Gaurish is going to use this boilerplate to create storyclub repository.

Reply:
Yeah, that makes sense. I have made changes in the latest commit.